packages/pigeon/platform_tests/test_plugin/linux/pigeon/core_tests.gen.h

```diff
@@ -1433,6 +1433,23 @@ G_DECLARE_FINAL_TYPE(CoreTestsPigeonTestMessageCodec,
                      CORE_TESTS_PIGEON_TEST, MESSAGE_CODEC,
                      FlStandardMessageCodec)
 
+const uint8_t core_tests_pigeon_test_an_enum_type_id = 129;
```

Comment:
I haven't done a full review, but from just skimming the generated code changes:
- Why are these `uint8_t` when the ID is an `int` in the `FlValue` headers?
- Constants should be declared (as `extern`) in a header, and defined in the implementation file, rather than have copies inlined into every file including this header.